src/ccall.cpp

```diff
@@ -87,11 +88,12 @@ static Value *runtime_sym_lookup(
     //       *llvmgv = jl_load_and_lookup(f_lib, f_name, libptrgv);
     //   }
     //   return (*llvmgv)
+    auto T_pjlvoidfunc = JuliaType::get_pjlvoidfunc_ty(irbuilder.getContext());
     BasicBlock *enter_bb = irbuilder.GetInsertBlock();
     BasicBlock *dlsym_lookup = BasicBlock::Create(irbuilder.getContext(), "dlsym");
     BasicBlock *ccall_bb = BasicBlock::Create(irbuilder.getContext(), "ccall");
-    Constant *initnul = ConstantPointerNull::get((PointerType*)T_pvoidfunc);
-    LoadInst *llvmf_orig = irbuilder.CreateAlignedLoad(T_pvoidfunc, llvmgv, Align(sizeof(void*)));
+    Constant *initnul = ConstantPointerNull::get(T_pjlvoidfunc);
+    LoadInst *llvmf_orig = irbuilder.CreateAlignedLoad(T_pjlvoidfunc, llvmgv, Align(sizeof(void*)));
```

Comment:
Was the rewrite from `T_pvoidfunc` to `T_pjlvoidfunc` intentional?

Reply:
It was, since I was ripping out all of the global types I decided to take the opportunity to try to standardize some of the naming (although I might have interpreted the meanings of some of them wrongly). Full list of renames:

jl_array_llvmt -> T_jlarray
pjl_array_llvmt -> T_pjlarray
jl_func_sig -> T_jlfunc
jl_func_sig_sparams -> T_jlfuncparams
T_pvoidfunc -> get_pjlvoidfunc_ty()